composer/packages/theme/src/definitions/views/diagram.less

```diff
@@ -39,6 +39,13 @@
                             text-anchor: middle;
                         }
                     }
+                    .dropdown-trigger {
+                        .dropdown {
+                            .trigger-icon {
+                                font-family: "font-ballerina"
```

Comment:
We'll have to add font to each icon seperately? Can't we add "font-ballerina" to the global font-family?